c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -2557,6 +2566,9 @@ namespace TransformStream {
    */
   bool constructor(JSContext* cx, unsigned argc, Value* vp) {
     CallArgs args = CallArgsFromVp(argc, vp);
+    if (!ThrowIfNotConstructing(cx, args, "TransformStream")) {
+      return false;
+    }
```

Comment:
With my `CTOR_HEADER` suggestion, this would become
```suggestion
    CTOR_HEADER("TransformStream", 0)
```